app/com/linkedin/drelephant/tunin/JobTuningInfo.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.linkedin.drelephant.tunin;
+
+import models.TuningParameter;
+import models.JobDefinition;
+import java.util.List;
+
+/**
+ * This class holds the tuning information for the job.
+ */
+public class JobTuningInfo {
+
+  private JobDefinition _tuningJob;
+
+  private String _tunerState; //archive, prev_population, current_population, rnd_state
```

Comment:
cleanup?

Reply:
I wanted to keep it as a note. I have modified the comment accordingly. Please check.